PROTOCOL.md

```diff
@@ -965,6 +966,27 @@ class SetDirtyStateAction implements Action {
 ```
 </details>
 
+### 2.6.3. ExportSvgAction
+
+The client (or the server) sends a `ExportSvgAction` to indicate that the diagram, which represents the current model state, should be exported in SVG format. The action only provides the diagram svg as plain string. The expected result of executing a `ExportSvgAction` is new file in SVG-format on the underlying filesystem. However, other details like the target destination, concrete file name, file extension etc. are not specified in the protocol. So it is the responsibility of the action handler to process this information accordingly and export the result to the underlying filesystem. 
```

Comment:
Minor typos:
- "The expected result of executing a `ExportSvgAction` is new file [...]" ->  "The expected result of executing a `ExportSvgAction` is a new file [...]" (missing 'a ')
- I would write an `ExportSvgAction` (adding the 'n')

Reply:
Thanks. Agreed. FYI, you can use the Suggestion feature of Github for (ctrl+g) for such "minor" improvements. This allows auto comiting of your suggestion.

Reply:
Done